docs/integrations/visual-context-system/scripts/terminal-integration/prompt-injector.sh

```
@@ -0,0 +1,97 @@
+#!/bin/bash
+# AIOS Terminal Integration - Prompt Injector
+# Customizes PS1 to show session context inline
+# Performance target: <5ms overhead per prompt
+
+# ANSI color codes
+CYAN='\033[0;36m'
+YELLOW='\033[1;33m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
```

Shellcheck (0.11.0)

[warning] RED appears unused. Verify use (or export if used externally).

(SC2034)